airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -43,8 +43,8 @@ def get_dag_state_info():
                 DagRun.state,
                 func.count(DagRun.state).label("count"),
             )
-            .group_by(DagRun.dag_id, DagRun.state)
-            .subquery()
+                .group_by(DagRun.dag_id, DagRun.state)
+                .subquery()
```

Comment:
the formatting feels off in all of these changes?

Reply:
There were automatically by my IDE, I have changed them